app/main/views/users.py

```diff
@@ -51,6 +53,26 @@ def get_user_by_id(user_id):
     return jsonify(users=user.serialize())
 
 
+@main.route('/teammembers/<string:domain>', methods=['GET'])
+def get_teammembers(domain):
+    userlist = User.query.filter_by(role='buyer')
+    teammembers = [_ for _ in userlist if _.viewrow().email_domain == domain]
+
+    query = db.session.execute("""
+        select * from govdomains
+        where domain = :domain
+    """, {'domain': domain})
```

Comment:
this method does anti-sql-injection escaping (not that we'd be trying to inject ourselves from our own csv files)